plugwise/util.py

```diff
@@ -93,6 +94,11 @@ def check_heater_central(xml: etree.Element) -> str:
         if heater_central.find("name").text == "Central heating boiler":
             hc_list.append({hc_id: has_actuators})
 
+    if not hc_list:
+        raise DataMissingError(
+            "No Central heating boiler found, please create an Issue"
+        )  # pragma: no cover
```

Comment:
Thats a bit vague - as we probably re-raise this from the front-end, I'd suggest we make a BoilerDataMissingError (and don't care too much about the actual message since we don't necessarily raise user errors from the supporting lib

Reply:
Agreed, I'll let it return NONE then we can detect that someplace else if we really want that.